services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/service.py

```diff
@@ -158,6 +159,7 @@ def from_scheduler_data(
             boot_type=ServiceBootType.V2,
             user_id=scheduler_data.user_id,
             project_id=scheduler_data.project_id,
+            # type: ignore
```

Comment:
what is the point of that one?

Reply:
it complains for the fields below. They are not found.